business/istio_config.go

```diff
@@ -86,14 +86,10 @@ func (icc IstioConfigCriteria) Include(resource string) bool {
 		return icc.IncludePolicies && !isWorkloadSelector
 	case kubernetes.MeshPolicies:
 		return icc.IncludeMeshPolicies && !isWorkloadSelector
-	case kubernetes.ServiceMeshPolicies:
-		return icc.IncludeServiceMeshPolicies && !isWorkloadSelector
```

Comment:
I think it would be good to remove the 
```
	criteria.IncludeServiceMeshPolicies = defaultInclude
	criteria.IncludeServiceMeshRbacConfigs = defaultInclude
```
From the criteria, also there are some pending models:
```
~/Projects/gopath/src/github.com/kiali/kiali(pr3256) $ grep -r "ServiceMeshPolicies" *
business/istio_config.go:       IncludeServiceMeshPolicies    bool
business/istio_config.go:       criteria.IncludeServiceMeshPolicies = defaultInclude
Coincidencia en el fichero binario _output/docker/kiali
~/Projects/gopath/src/github.com/kiali/kiali(pr3256) $ grep -r "ServiceMeshRbacConfigs" *
business/istio_config.go:       IncludeServiceMeshRbacConfigs bool
business/istio_config.go:       criteria.IncludeServiceMeshRbacConfigs = defaultInclude
models/cluster_rbac_config.go:type ServiceMeshRbacConfigs []ServiceMeshRbacConfig
models/cluster_rbac_config.go:func (rcs *ServiceMeshRbacConfigs) Parse(smRbacConfigs []kubernetes.IstioObject) 
```

That you'd like to clean.